restler/unit_tests/log_baseline_test_files/test_replay.txt

```diff
@@ -0,0 +1,7 @@
+-> PUT /B/B HTTP/1.1\r\nAccept: application/json\r\nHost: unittest\r\nContent-Type: application/json\r\nAuthorization: valid_unit_test_token\r\n\r\n

```

Comment:
>PUT [](http://example.com/codeflow?start=3&length=3)

Could you please rename this file to 'replay_sanity_test_for_tracedb.replay.txt'?